### PR TITLE
Document how to deploy the editor to the AWS development environment

### PR DESCRIPTION
**Pull request by mjgiarlo**
Fixes LD4P/sinopia#209

This can be run from any branch at any time to refresh the dev instance of the editor.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -107,15 +106,45 @@ $ docker-compose up platformdata # add the '-d' flag to daemonize and run in bac
...
+Then push the `dev`-tagged issue to DockerHub:
+
+```shell
+$ docker push ld4p/sinopia_editor:dev
+```
+
+Next, set an environment variable to the name of the DLSS AWS dev profile as described in the documentation above:
+
+```shell
+$ export AWS_PROFILE=development
```
Comment (jgreben) [inline README.md]: I had to set this to something other than `development`. Maybe add "Check for the DevelopersRole profile that you are using as configured in `~/.aws/config`"?

File: README.md [hunk 2]
```diff
@@ -107,15 +106,45 @@ $ docker-compose up platformdata # add the '-d' flag to daemonize and run in bac
...
+```
+
+And, finally, run the following `aws` CLI commands to refresh the dev ECS instance that runs the editor:
```
Comment (jgreben) [inline README.md]: At first I thought I had to prepend `aws` to all of these commands but then noticed that the first two were just setting variables. Maybe reword as "finally, set the task_arn and cluster_arn variables, and then run the following `aws` CLI command to refresh...."

File: README.md [hunk 3]
```diff
@@ -107,15 +106,45 @@ $ docker-compose up platformdata # add the '-d' flag to daemonize and run in bac
...
+Ask a member on the DevOps team to go into the AWS console to update https://sinopia.io
+
+### Updating Docker Image in AWS Dev Environment
+
+This section assumes you've already authenticated to DockerHub via `docker login` in the previous section, and also assumes you've run through the [AWS development environment setup](https://github.com/sul-dlss/terraform-aws/wiki/AWS-DLSS-Dev-Env-Setup) documentation and configured the AWS CLI.
+
+First, build a new `sinopia_editor` image tagged with `dev`. In order to do this, you **MUST** provide the dev-specific build args:
+
+```shell
+$ docker build -t ld4p/sinopia_editor:dev --build-arg SPOOF_SINOPIA_SERVER=false --build-arg TRELLIS_BASE_URL=https://trellis.development.sinopia.io --build-arg DEFAULT_PROFILE_SCHEMA_VERSION=0.0.2 --build-arg SINOPIA_GROUP=ld4p --build-arg SINOPIA_URI=https://development.sinopia.io --build-arg AWS_COGNITO_DOMAIN=https://sinopia-development.auth.us-west-2.amazoncognito.com --build-arg COGNITO_CLIENT_ID=2u6s7pqkc1grq1qs464fsi82at .
```
Comment (jgreben) [inline README.md]: These env vars all seem ok to me to have published here.

Comment (jermnelson) [reply README.md]: 👍

File: README.md (hunk 1 above)
Comment (mjgiarlo) [reply README.md]: Oh, I meant to make this change. Good catch!

Comment (mjgiarlo) [reply README.md]: Done.

File: README.md (hunk 2 above)
Comment (mjgiarlo) [reply README.md]: Done!

File: README.md (hunk 3 above)
Comment (mjgiarlo) [reply README.md]: thx!
